static/src/js/reducers/collectionMetadata.js

```diff
@@ -13,8 +17,31 @@ const initialState = {
   projectIds: []
 }
 
+const processResults = (results) => {
```

Comment:
pull out into a util and update all reducers that use it, collections and granules?